Update Encounter.cs

Corlobin opened the pull request:
Rollback. POGOProtos is using long for this encounterID, I did a pull request at him repository.

martin-podlubny commented:
Should review/merge the first PR I think :)

Maufeat commented:
Executed that EvolveAllGivenPokemons, put all my pokemons in, and it tried to evlove my Voltorb the whole time but it can't (not enough candys) but it says it is successfully evolved in a loop. 

Like this:

> Evolved V0100PokemonVoltorb successfully for 0xp
Evolved V0100PokemonVoltorb successfully for 0xp
Evolved V0100PokemonVoltorb successfully for 0xp
Evolved V0100PokemonVoltorb successfully for 0xp
Evolved V0100PokemonVoltorb successfully for 0xp

martin-podlubny commented:
@Maufeat Don't know, haven't really try evolving yet :D Fix it, I don't have time ATM :)

Maufeat commented:
Oki

martin-podlubny commented:
Using this in https://github.com/FeroxRev/Pokemon-Go-Rocket-API/pull/53

Corlobin committed:
Update Encounter.cs

Rollback. POGOProtos is using long for this encounterID, I did a pull request at him repository.
